packages/persist/lib/server.integration.test.ts

```diff
@@ -3,8 +3,8 @@ import { server } from './server.js';
 import fetch from 'node-fetch';
 import type { AuthCredentials, Connection, Sync, Job as SyncJob, Environment, Account } from '@nangohq/shared';
 import db, { multipleMigrations } from '@nangohq/database';
-import { createActivityLog, environmentService, connectionService, createSync, createSyncJob, SyncType, SyncStatus, accountService } from '@nangohq/shared';
-import { logContextGetter } from '@nangohq/logs';
+import { environmentService, connectionService, createSync, createSyncJob, SyncType, SyncStatus, accountService } from '@nangohq/shared';
+import { logContextGetter, migrateMapping } from '@nangohq/logs';
```

Comment:
what about`migrateMapping as migrateLogsMapping` so when you read the code without the import it is clearer what mappings we are talking about?

Reply:
true